FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,123 @@
+        if arg_and_value.size() != 2:
+            return
+
+        map_override = arg_and_value[1]
+
+        func_godot_map.local_map_file = map_override
+        func_godot_map.verify_and_build()</code></pre>
+    <h2>Setting Up Your Editor</h2>
+    <p>In the below guides, there will be references made to executables. This is up to the user, however you can use the Godot executable to run your game straight from the editor, or you could use exported binaries of your game. On Windows, your Godot Binary path is where you keep your Godot.exe file. On most Linux systems this is found under <code>/usr/bin/godot</code>. An example executable path and parameters for running a game in the editor: <code class="highlight">C:/GAMEDEV/Godot.exe --path C:/func_godot_project_example/ -- run_map::${MAP_BASE_NAME}.map</code>. Note that <code class="highlight">--path</code> is to the directory where project.godot resides. See the full list of Godot engine <a href="https://docs.godotengine.org/en/stable/tutorials/editor/command_line_tutorial.html" target="_blank">Command Line Interface options</a>.</p>
+    <h3>TrenchBroom</h3>
+    <p>
+        The TrenchBroom documenation <a href="https://trenchbroom.github.io/manual/latest/#launching_game_engines">covers this generally</a>.<br>
+
+        To run your map from TrenchBroom, navigate to <code class="highlight">Run > Launch Engine</code>.
+        Go to <code class="highlight">Configure engines...</code>, add a game engine profile, and provide a name and path to the executable for this. For this example, we use the path to the Godot editor executable.
+        Once you're happy with your engine configuration, click close. You will now see this as an option you can select. Select it.
+        <p align="center"><img src="../images/tips_editorbuild_trenchbroom1.png"><br></p>
+        Since we're using the Godot editor, as a parameter need to provide a path to the Godot project. We then mark user-made options with <code class="highlight">--</code>. In here, we use our <code class="highlight">run_map::</code> parameter, using the built in TrenchBroom variable <code class="highlight">${MAP_BASE_NAME}</code>. We follow this with <code class="highlight">.map</code> as this name is without the file extension.
+    </p>
+
+    <h3>NetRadiant Custom</h3>
+
+    <p>
+        NetRadiant itself provides some basic scripting for this process, defined in the gamepack folder. 
+
+        The path to our default build profiles can be found in your NetRadiant gamepack folder. This is something defined in your 
+        <a href = https://func-godot.github.io/func_godot_docs/FuncGodot%20Manual/pages/ref_netradiant_custom_resources.html#GamepackConfig>NetRadiantCustomGamePackConfig</a>.
+
+        If you haven't set this up, you'll need to first complete <a href = https://func-godot.github.io/func_godot_docs/FuncGodot%20Manual/pages/guide_map_editor_config.html#NetRadiantCustom>creating your NetRadiantCustomGamePackConfig.</a>
```

Comment:
Can be deleted. It is safe to assume if the user has gotten this far they've already set up their gamepack in the earlier setup chapter, just as we assume TrenchBroom users have set up their game configs already.